lib/player.js

```diff
@@ -1166,6 +1166,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         this.adManager_.onAssetUnload();
       }
 
+      if (this.video_) {
+        // Remove all track nodes
+        while (this.video_.lastChild) {
```

Comment:
We have a utility function that does this, incidentally.
`shaka.util.Dom.removeAllChildren`

Reply:
Changed!